src/defs/metrics.d.ts

```diff
@@ -297,32 +327,39 @@ declare module "tsdDef" {
     export interface MetricsEvent {
         /**
          * The annotations represented as hash of arrays indexed by annotation name.
-         * @type {{Object.<string, MetricsList>}}
          */
-        annotations:{[name:string]: string};
+        annotations:Annotations2F;
 
         /**
          * Counters and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         counters:{[name:string]: MetricsList<MetricSample>};
 
         /**
          * Gauges and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         gauges:{[name:string]: MetricsList<MetricSample>};
 
         /**
          * Timers and their samples recorded represented as hash of counter name to
          * [MetricSample]{@linkcode MetricSample}
-         * @type {{Object.<string, MetricsList>}}
          */
         timers:{[name:string]: MetricsList<MetricSample>};
     }
 
+    export interface Annotations2F {
+        // A date-time. Must be formatted according to RFC 3339, section 5.6. See https://tools.ietf.org/html/rfc3339#section-5.6
+        _end: string;
+        // A date-time. Must be formatted according to RFC 3339, section 5.6. See https://tools.ietf.org/html/rfc3339#section-5.6
```

Comment:
This is output specific; e.g. as defined by the json schema. Inside the library this should be whatever conventionally represents a date-time. Granted, the Java client does not do this well at the moment. The Event class published from metrics on close to the client sinks should contain the timestamp, annotations, dimensions, and samples. 

https://github.com/ArpNetworking/metrics-client-java/blob/master/src/main/java/com/arpnetworking/metrics/Event.java

I also don't think we should call out specific annotation values.